docs/guides/implement-a-custom-logger.md

```diff
@@ -73,7 +73,7 @@ const runtime = new Runtime({
 });
 ```
 
-Start the applications and open the dev tools. Refresh the page. The console logs should be displayed:
+Start the applications and open the developer tools. Then, refresh the page. You should see the following console log message:
```

Comment:
> Start the applications and open the developer tools. Then, refresh the page

we could merge those 2 sentences.


Start the applications and open the developer tools. then refresh the page